docs/maps/maps-aggregations.asciidoc

```diff
@@ -68,9 +68,9 @@ To enable a blended layer that dynamically shows clusters or documents:
 
 [role="xpack"]
 [[maps-top-hits-aggregation]]
-=== Top hits per entity
+=== Display the most relevant entities
```

Comment:
How about "Display the most relevant documents per entities"? The layer is not displaying entities but the top documents per entity.

Reply:
Is it possible to come up with a more specific word for entity?

Reply:
I like Nathan's suggestion; I think we should use entity.

Reply:
++